src/system.c

```diff
@@ -848,11 +850,11 @@ int lseek( int file, int ptr, int dir )
  * @param[in] flags
  *            Flags specifying open flags, such as binary, append.
  * @param[in] mode
- *            Mode of the file.
+ *            Mode of the file (currently ignored).
  *
  * @return File handle to refer to this file on success, or a negative value on error.
  */
-int open( char *file, int flags, int mode )
+int open( const char *file, int flags, ... )
```

Comment:
For reference:
https://sourceware.org/git/?p=newlib-cygwin.git;a=blob;f=newlib/libc/syscalls/sysopen.c;hb=main
`man 2 open`
`/opt/libdragon/mips64-elf/include/sys/_default_fcntl.h`